examples/shopping_ads/get_product_category_constant.rb

```diff
@@ -63,17 +61,17 @@ def get_product_bidding_category_constant(customer_id)
   root_categories = Set.new
 
   response.each do |row|
-    product_bidding_category = row.product_bidding_category_constant
+    product_category = row.product_category_constant
     category = {
-      name: product_bidding_category.localized_name,
-      id: product_bidding_category.resource_name,
+      name: product_category.localizations,
```

Comment:
The `localizations` field is a repeated field, so you need to find the `en`/`US` entry in that collection

Reply:
Done

Reply:
Would `select` followed by `first` be more idiomatic for Ruby?

Reply:
I usually try to keep those a bit more generic but I changed it to `detect` which i think is more idiomatic.

Reply:
Thanks! The updated version looks easier to read.